planner/core/stats.go

```diff
@@ -202,6 +202,7 @@ func (ds *DataSource) getGroupNDVs(colGroups [][]*expression.Column) []property.
 					Cols: idxCols,
 					NDV:  float64(idx.NDV),
 				}
+				logutil.BgLogger().Warn("get group ndv", zap.Int64s("cols", idxCols), zap.Float64("ndv", ndv.NDV))
```

Comment:
It seems a debug record, should we remove it?

Reply:
Yes, will be removed when merge master or address other comments.